wordpress-mobile/WordPress-Android
# UI - Implement personalization screen for quick links

Part of #https://github.com/wordpress-mobile/WordPress-Android/issues/19270

## Description
This PR implements the personalization screen for shortcuts

## 📷  Screenshot
|  Light | Dark   | 
|---|---|
|   ![Screenshot_20230927_185104](https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/0c2fd414-33ac-4d14-bac4-21140d226aa7)| ![Screenshot_20230927_185126](https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/782217af-de64-4ede-a54b-bf8e065bacc6)  |   


## To test:
1. Go to jetpack  app 
2. Go to dashboard 
3. Click on personalize your home tab
4. Go to shortcuts
5. Verify that all the site items are shown in shortcuts

## Regression Notes
1. Potential unintended areas of impact
Nothing as this is a new screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Unit tests

PR submission checklist:

UI Changes testing checklist:
